packages/react/src/themes/teams/components/Label/labelVariables.ts

```diff
@@ -16,11 +14,17 @@ export interface LabelVariables {
 export default (siteVars: SiteVariablesPrepared): LabelVariables => {
   const color = 'rgba(0, 0, 0, 0.6)'
 
+  const colorScheme = {
+    ...siteVars.colorScheme,
+    default: {
+      ...siteVars.colorScheme.default,
```

Comment:
isn't this a bit dangerous to do explicitly? maybe extract in a `mergeColors` function? (only if you think it's going to be reused/needed again)

Reply:
IMHO this is much explicit and clear instead of having a function that will do that, in the end this is just object spread of the default values.. However I see your point, and someone can easily wrongly define the defaults... @miroslavstastny what is your thought on this?

Reply:
Added colorUtils in the teams theme for handling this. Not sure if it should be exported somewhere from stardust though..